src/traces/choropleth/plot.js

```diff
@@ -151,11 +150,13 @@ plotChoropleth.style = function(geo) {
             var trace = calcTrace[0].trace,
                 s = d3.select(this),
                 marker = trace.marker || {},
-                markerLine = marker.line || {},
-                zmin = trace.zmin,
-                zmax = trace.zmax,
-                scl = getColorscale(trace.colorscale),
```

Comment:
Ah good catch, we've already done this at the `supplyDefaults` step so no need to repeat it anywhere else.